libs/client-shared/src/lib/components/button/button.component.scss

```
@@ -73,6 +73,11 @@
       color: variables.$white;
     }
   }
+
+  &:focus.asset-sg-active,
```

Pseudo-classes need to be written after classes (does this even work this way?):
```suggestion
  &.asset-sg-active:focus,
```

---

yep, worked 👯